### PR TITLE
Improved explanations of a solution to a challenge

### PR DESCRIPTION
**Pull request by Aleksandar15**
Improved solution to a challenge # 3. at [Challenges](https://react.dev/learn/javascript-in-jsx-with-curly-braces#challenges).

Current explanation:
current explanations

Improved explanation:
improved explanations

### CODE REVIEW

**Comment by gaearon** [comment]
I find this a bit too wordy — it makes it sounds more complex than it is. Can you think of a more concise explanation?

**Comment by Aleksandar15** [comment]
> I find this a bit too wordy — it makes it sounds more complex than it is. Can you think of a more concise explanation?

That was my initial intention, then I went on with the step-by-step guide because the whole section is beginner-friendly and keeps reminding users of the JavaScript's fundamentals, but this explanation was kinda "*neglected*".

I was also thinking of when:
a) More experienced users read this, the first sentence will be enough to give them *aha-moment* and scroll down to the code.
b) Beginners read this, still have no idea what's happening, and continue to read the steps to give them *aha-moment*.

A workaround I can think of is to keep the 1-sentence above the Sandbox, but move the steps below the Sandbox, and have it say "*Here's what happened:*"

What's your take on it? Should I rather shorten the steps, or move them below ?

**Comment by Aleksandar15** [comment]
Shortened explanation:

image
